http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/WsmApiClientProvider.scala

```diff
@@ -23,32 +24,42 @@ import org.typelevel.log4cats.StructuredLogger
  */
 trait WsmApiClientProvider[F[_]] {
 
+  // WSM state can be BROKEN, CREATING, READY, UPDATING or None, (deleted or doesn't exist)
   val possibleStatuses: Array[WsmState] =
-    State.values().map(_.toString).map(Some(_)).map(WsmState(_)) :+ WsmState(Some("DELETED"))
+    State.values().map(_.toString).map(Some(_)).map(WsmState(_)) :+ WsmState(Some("NONE"))
 
   def getControlledAzureResourceApi(token: String)(implicit ev: Ask[F, AppContext]): F[ControlledAzureResourceApi]
   def getResourceApi(token: String)(implicit ev: Ask[F, AppContext]): F[ResourceApi]
```

Comment:
did a bit of a refactor here to move the match case from the AppService here (in getWsmState) instead